backend/src/graphql/resolver/AdminResolver.ts

```diff
@@ -11,18 +16,161 @@ export class AdminResolver {
   async searchUsers(@Arg('searchText') searchText: string): Promise<UserAdmin[]> {
     const loginUserRepository = getCustomRepository(LoginUserRepository)
     const loginUsers = await loginUserRepository.findBySearchCriteria(searchText)
-    const users = loginUsers.map((loginUser) => {
-      const user = new UserAdmin()
-      user.firstName = loginUser.firstName
-      user.lastName = loginUser.lastName
-      user.email = loginUser.email
-      user.creation = [
-        (Math.floor(Math.random() * 50) + 1) * 20,
-        (Math.floor(Math.random() * 50) + 1) * 20,
-        (Math.floor(Math.random() * 50) + 1) * 20,
-      ]
-      return user
-    })
+    const users = await Promise.all(
+      loginUsers.map(async (loginUser) => {
+        const user = new UserAdmin()
+        user.firstName = loginUser.firstName
+        user.lastName = loginUser.lastName
+        user.email = loginUser.email
+        user.creation = await getUserCreations(loginUser.id)
+        return user
+      }),
+    )
     return users
   }
+
+  @Query(() => [Number])
+  async createPendingCreation(
+    @Args() { email, amount, note, creationDate, moderator }: CreatePendingCreationArgs,
+  ): Promise<number[]> {
+    const userRepository = getCustomRepository(UserRepository)
+    const user = await userRepository.findByEmail(email)
+
+    const creations = await getUserCreations(user.id)
+    const creationDateObj = new Date(creationDate)
+    if (isCreationValid(creations, amount, creationDateObj)) {
+      const pendingCreationRepository = getCustomRepository(PendingCreationRepository)
+      const loginPendingTaskAdmin = pendingCreationRepository.create()
+      loginPendingTaskAdmin.userId = user.id
+      loginPendingTaskAdmin.amount = BigInt(amount * 10000)
+      loginPendingTaskAdmin.created = new Date()
+      loginPendingTaskAdmin.date = creationDateObj
+      loginPendingTaskAdmin.note = note
+      loginPendingTaskAdmin.moderator = moderator
+
+      pendingCreationRepository.save(loginPendingTaskAdmin)
+    }
+    return await getUserCreations(user.id)
+  }
+}
+
+async function getUserCreations(id: number): Promise<number[]> {
```

Comment:
This one must be refactored. Can we make one call and group the amount by months and get the sums directly from the DB? May be not in this PR but these are too many DB calls by now.

Reply:
Like we discussed it: "Search all OpenCreations & StoredCreations of the last 3 months and groups them by the month name" This should create a table like following: 
September => Sum of creations for userId and date Contains september
October => Sum of creations for userId and date Contains october
November => Sum of creations for userId and date Contains november

Reply:
ToDo:
- [ ] Refactor the code so that we have 2 DB calls one for the OpenCreations and one for the StoredCreations and group them by the creation date.